src/Fabulous.AST.Tests/UnitTest1.fs

```diff
@@ -0,0 +1,22 @@
+module Fabulous.AST.Tests
+
+open NUnit.Framework
+
+open Fabulous.AST
+open type Fabulous.AST.Node
+
+[<Test>]
+let CanCompileBasicTree () =
+    let tree =
+        ModuleOrNamespace(
+            Range("test.fs", Position(1, 2), Position(3, 4))
+        )
+        
+    let rootNode = Tree.compile tree :?> Fantomas.Core.SyntaxOak.ModuleOrNamespaceNode
```

Comment:
I'm not sure if `compile` is a Fabulous jargon, but in this case, it sounds a bit confusing as the converted `Oak` to code can be compiled by F#.